### PR TITLE
Add new chain to givEconomy guide

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to https://github.com/Giveth/giveth-dapps-v2/issues/4723#issuecomment-2370621560

### CODE REVIEW

File: add-new-chain-to-givEconomy-guide.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+8. Enter this command to check everything is working locally
+
+`forge script script/deployTokenDistro.s.sol  --rpc-url  {rpcUrl}  --legacy --chain 2442 {Desired chainId}`
+
+
+9. Enter this command to broadcast and verify the smart contracts on blockchain
+
+`forge script script/deployTokenDistro.s.sol  --rpc-url  {rpcUrl}    --verify --broadcast --legacy --chain 2442 {Desired chainId}`
+
+9. Verify contract with proxy admin in the block explorer
```
Comment (aminlatifi) [inline add-new-chain-to-givEconomy-guide.md:76]:
```suggestion
9. Open the block-explorer website's smart contract page (code section) and declare it as a proxy.
<img width="278" alt="image" src="https://github.com/user-attachments/assets/a9dec69a-04ef-4807-b528-cfb3d9ce533b">

```

File: add-new-chain-to-givEconomy-guide.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
```
Comment (aminlatifi) [inline add-new-chain-to-givEconomy-guide.md:77]: Permission adding/revoking can be added as part of ownership transfer

File: add-new-chain-to-givEconomy-guide.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+7. Check the address of `DISTRIBUTOR_ROLE` in tokenDistro contract https://cardona-zkevm.polygonscan.com/address/0x2df3e67be4e441cddd2d29c3d41dfd7d516f18e6#readProxyContract
+
+8. Call `grantRole` in `tokenDistro` https://cardona-zkevm.polygonscan.com/address/0x2df3e67be4e441cddd2d29c3d41dfd7d516f18e6#writeProxyContract and grant `DISTRIBUTOR_ROLE` to givPower unipool address
+
+9. Call `assign()` method in tokenDistro and assign the reward token amount to unipool givPower
+
+
+## Test
+1. Call `stake` method in unipoolGivPower contract
+2. See the giviverse expansion percentage in https://giveth.io/givstream
```
Comment (aminlatifi) [inline add-new-chain-to-givEconomy-guide.md:115]: giviverse?!

File: add-new-chain-to-givEconomy-guide.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+## Integrate with Frontend
+You can know how to do it with looking at below PR
+https://github.com/Giveth/giveth-dapps-v2/pull/4562/files
+(LM adress is the UnipoolGivPower address, for staging ENV you should add configs to `development.tsx` and for produciton you need to add the configs to `production.tsx`
+
+## Notify Reward Bot
+1. We have a wallet address that is in charge for this purpose, we have th private key of that address in the server, so we should make sure call`setRewardDistribution` on Unipool GivPower to set the wallet address of this private key as `reward distributor`
+2. We write a script in `giv-token-contract` to call it with crontab jobs https://github.com/Giveth/giv-token-contracts/commit/5188586beed3e0f73c7e0b9bbaf653134184a215
+3. We should transfer some monet the reward distributor address for the gas fee on the new chain that we want ( if reward distributor doesnt have gas, the no`notifyRewardAmount` tx can't be executed
```
Comment (aminlatifi) [inline add-new-chain-to-givEconomy-guide.md:129]:
```suggestion
3. Transfer some gas to the reward distributor address if it doesn't have enough to make `notifyRewardAmount` transactions
```

File: add-new-chain-to-givEconomy-guide.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+4. Setup the crontab on the server to execute this script periodically
+
+## Giv unlcok bot
+1. Copy a folder of https://github.com/Giveth/givpower-bot in the server, then change config values to the new chain and bring up the docker compose, then this app will unlcok the GIVs when it's the time
+2. Make sure that the wallet address of the private key that we put in the config have enough gas fee in it
```
Comment (aminlatifi) [inline add-new-chain-to-givEconomy-guide.md:134]:
```suggestion
2. Make sure that the wallet address of the private key that we put in the config has enough gas in it
```

File: add-new-chain-to-givEconomy-guide.md (hunk 3 above)
Comment (mohammadranjbarz) [reply add-new-chain-to-givEconomy-guide.md:117]: Screenshot 1403-07-16 at 1 12 16 in the afternoon
Yes

**Pull request merged**